### ISSUE TITLE
Bug: NPM auth stopped working (PRO)

### ISSUE THREAD

**Issue by zamson** [FortAwesome/Font-Awesome#19822]
### Bug description

Yesterday afternoon we started seeing build errors on our websites:

```
npm ERR! code E401
npm ERR! Incorrect or missing password.
npm ERR! If you were trying to login, change your password, create an
npm ERR! authentication token or enable two-factor authentication then
npm ERR! that means you likely typed your password in incorrectly.
npm ERR! Please try again, or recover your password at:
npm ERR!     https://www.npmjs.com/forgot
npm ERR! 
npm ERR! If you were doing some other operation then your saved credentials are
npm ERR! probably out of date. To correct this please try logging in again with:
npm ERR!     npm login
```

We have not changed anything on our end and it's been working for years. Even after regenerating the key and updating .npmrc it still does not work (multiple projects failing to auth). Bill has been paid since multiple days (recently CC got declined since the card was blocked by the bank). All configurations has been confirmed, it something on your end.

I'm in contact with support as well - posting here so you can figure what happened and report back. Our client's website contain time sensitive information and not being able to deploy is a huge problem.

## So to summarize

- Auth is not working on the private NPM repo even though  bill has been paid after CC was declined.

### Font Awesome version

6.0.0

### Serving

Other (as specified in the bug description)

### Implementation

Other (as specified in the bug description)

### Browser and Operating System

- MAC OS, Netlify, Vercel

### Web bug report checklist

- [X] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by deathnfudge** [comment]
Hi,

Did you get this resolved with support? If not, please reach out again so we can get you fixed up.

**Comment by zamson** [comment]
@deathnfudge Yeah, finally got resolved - thanks. I suspect it was because one payment failed. Would be great if steps could be taken so this does not happen again.

I switched to running a local package on some of the more sensitive websites.

**Comment by deathnfudge** [comment]
Good deal. We've got a fix in so it won't happen going forward.
